src/brogue/Time.c

```diff
@@ -989,10 +989,20 @@ void playerFalls() {
         rogue.depthLevel++;
         startLevel(rogue.depthLevel - 1, 0);
         damage = randClumpedRange(FALL_DAMAGE_MIN, FALL_DAMAGE_MAX, 2);
-        messageWithColor("You are damaged by the fall.", &badMessageColor, 0);
-        if (inflictDamage(NULL, &player, damage, &red, false)) {
-            gameOver("Killed by a fall", true);
-        } else if (rogue.depthLevel > rogue.deepestLevel) {
+        boolean killed = false;
+        if (terrainFlags(player.xLoc, player.yLoc) & T_IS_DEEP_WATER) {
+            messageWithColor("You fall into deep water, unharmed.", &badMessageColor, 0);
```

Comment:
plunge?  "fall" feels a little overused in nearby messaging.

Reply:
Nice one! Also, I'd propose "You are ~~damaged~~ _injured_ by the fall".

Reply:
"plunge" is used a lot for the previous falling message (also all nearby message calls are alternatives so it's not overused in the actual game time). So I prefer fall. But it's minor so I can merge and we can see it in action?

Reply:
Or: dive, splash, hit